images/ig-publisher-localdev/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ghcr.io/fhir/ig-publisher-base:latest
```

Comment:
All this could also be moved to its parent Dockerfile and then keep it as a multi stage instead

Reply:
Done, thanks for the suggestion!